src/Appwrite/Task/SDKs.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+namespace Appwrite\Task;
+
+use Utopia\Platform\Action;
+use Utopia\Config\Config;
+use Utopia\CLI\Console;
+use Appwrite\Spec\Swagger2;
+use Appwrite\SDK\SDK;
+use Appwrite\SDK\Language\CLI;
+use Appwrite\SDK\Language\PHP;
+use Appwrite\SDK\Language\Web;
+use Appwrite\SDK\Language\Node;
+use Appwrite\SDK\Language\Python;
+use Appwrite\SDK\Language\Ruby;
+use Appwrite\SDK\Language\Dart;
+use Appwrite\SDK\Language\Deno;
+use Appwrite\SDK\Language\DotNet;
+use Appwrite\SDK\Language\Flutter;
+use Appwrite\SDK\Language\Go;
+use Appwrite\SDK\Language\Kotlin;
+use Appwrite\SDK\Language\Android;
+use Appwrite\SDK\Language\Swift;
+use Appwrite\SDK\Language\SwiftClient;
+use Exception;
+use Throwable;
+
+class SDKs extends Action
+{
+    public const NAME = 'sdks';
+
+    public function __construct()
+    {
+        $this
```

Comment:
Let's make sure we now have a description for all tasks.

Reply:
done